src/pfe/portal/routes/projects/metrics.route.js

```diff
@@ -68,11 +68,11 @@ router.get('/api/v1/projects/:id/metrics/status', checkProjectExists, async func
     const project = getProjectFromReq(req);
     const { canMetricsBeInjected, appStatus } = project;
     let capabilities = project.getMetricsCapabilities();
-    if (!capabilities || Object.keys(capabilities).length === 0) {
-      // If projectMetrics is an empty object, fetch them first
-      const { capabilities: updatedCapabilities } = await project.setMetricsState();
-      capabilities = updatedCapabilities;
-    }
+    //if (!capabilities || Object.keys(capabilities).length === 0) {
```

Comment:
These lines should probably be removed not commented out.
Also this means that capabilities is assigned on line 70, then reassigned on line 74 without being read. Line 70 could also be deleted. (Assuming `project.getMetricsCapabilities()` has no side effects.)

Reply:
ok, modified those lines